### PR TITLE
only return surveyResponses in current user's org

### PR DESCRIPTION
**Pull request by ericcater**
- Does not protect findUniqueSurveyResponse right now (nor any other findUnique) beyond requiring a logged in user.

### CODE REVIEW

File: apps/ods-api/src/surveyResponse/surveyResponse.service.ts [hunk 1]
```diff
@@ -112,48 +50,48 @@ export class SurveyResponseService {
   }
 
   async delete(
-    orgWhereUniqueInput: Prisma.SurveyResponseWhereUniqueInput
+    surveyResponseWhereUniqueInput: Prisma.SurveyResponseWhereUniqueInput
   ): Promise<{ deleted: boolean; message?: string }> {
     try {
       await this.prisma.surveyResponse.delete({
-        where: orgWhereUniqueInput,
+        where: surveyResponseWhereUniqueInput,
```
Comment (yurisim) [inline apps/ods-api/src/surveyResponse/surveyResponse.service.ts:57]: Good catch :)

File: apps/ods-api/src/surveyResponse/surveyResponse.service.ts [hunk 2]
```diff
@@ -112,48 +50,48 @@ export class SurveyResponseService {
...
+        ...surveyResponseWhereInput,
       },
     });
 
-    return {
-      unresolved: unresolvedCount,
-      overdue: overdueCount,
-      resolved: resolvedCount,
-    };
+    return Promise.all([unresolvedCount, overdueCount, resolvedCount]);
```
Comment (yurisim) [inline apps/ods-api/src/surveyResponse/surveyResponse.service.ts:94]: This is better! Nice.

**Review by yurisim — CHANGES_REQUESTED** [review]
Don't think the survey responses are being shown

File: apps/ods-api/src/surveyResponse/surveyResponse.service.ts (hunk 2 above)
Comment (ericcater) [reply apps/ods-api/src/surveyResponse/surveyResponse.service.ts]: Changed it again, I think the newest way is best

File: apps/ods-api/src/surveyResponse/surveyResponse.resolver.ts [hunk 3]
```diff
@@ -27,6 +29,7 @@ export class SurveyResponseResolver {
     return this.surveyResponseService.findMany({});
   }
 
+  //TODO findUnqiue is called from frontend, not sure how to prevent commanders from looking at other orgs' responses
```
Comment (yurisim) [inline apps/ods-api/src/surveyResponse/surveyResponse.resolver.ts:32]: find findUnqiue :(

**Review by yurisim — CHANGES_REQUESTED** [review]
Please merge dev

**Comment by ericcater** [comment]
> Please work with @Mikehickey7 to verify that responses show up in certain profiles and NOT others using UUIDs.
> 
> This is part of issue 155. It's marked as "done" on Jira but this issue (154) is part of it.

TGYTFYTYF

**Review by yurisim — DISMISSED** [review]

**Review by yurisim — APPROVED** [review]

File: apps/ods/src/app/survey-questions/survey-questions.component.ts [hunk 4]
```diff
@@ -89,7 +89,9 @@ export class SurveyQuestionsComponent implements OnInit, OnDestroy {
 
     // TODO: Nested behaviors like this are hard to test.
     this.surveyService
-      .submitWithQuestions(this.questions)
+      .submitWithQuestions(this.questions, {
+        name: '552 ACNS',
+      })
       .subscribe(({ data }) => {
```
Comment (yurisim) [inline apps/ods/src/app/survey-questions/survey-questions.component.ts:95]: may need to revert this change. Why did we do this again? Goni doesn't remember.

Comment (yurisim) [inline apps/ods/src/app/survey-questions/survey-questions.component.ts:95]: may need to revert this change. Why did we do this again? Goni doesn't remember.

**Pull request merged**